src/ipc/ipc-common/IPCProxyModelProperty.h

```diff
@@ -30,6 +30,9 @@
 #pragma once
 
 #include <QObject>
+#include <QtDBus>
```

Comment:
no DBus stuff in that header please.

Reply:
The same here, what is the rationale for it? The facelift is based on DBus.

Reply:
Your understanding that Facelift is based on DBus is wrong. It happens to use DBus for its IPC layer currently, but it could very well use another kind of channel. There has been some work on using another IPC in the past.
We do not want any DBus stuff inside the "ipc-common" folder.